packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkConfiguration.kt

```diff
@@ -22,6 +22,7 @@ package com.datadog.reactnative
  * @param trackFrustrations Whether to track frustration signals or not.
  * @param uploadFrequency The frequency to which batches of data are sent (can be 'RARE', 'AVERAGE' (default), 'FREQUENT')
  * @param batchSize The preferred size for uploaded batches of data (can be 'SMALL', 'MEDIUM' (default), 'LARGE')
+ * @param trackBackgroundEvents Whether to track events when the app is in background.
```

Comment:
I think we need to explain it a bit better.

Android documentation says the following:

```
         * Enables/Disables tracking RUM event when no Activity is in foreground.
         *
         * By default, background events are not tracked. Enabling this feature might increase the
         * number of sessions tracked and impact your billing.
```

so we need to highlight impact on the billing, and as well tell that by background here we mean when there is no active screen: it means generally speaking that some event happening when there is no active view (say because there is a mistake in the instrumentation), but when application is still in foreground can be considered as background event.

Reply:
Good point. I'll update all documentation parts about this.